config/logs.txt

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Do we really need the logs file to be pushed?

Reply:
I think logs.txt need not be in the PR